### DISCUSSION TITLE
Introducing Learning Paths on Global Campus

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#64543] [GitHub Education]
![NUX_Octodex](https://github.com/community/community/assets/107881423/57e515c5-5a64-459c-95eb-24609681e12f)

### All About Learning Paths

New to GitHub and still navigating your first days here on Global Campus? Or maybe you’re returning to start a new semester this fall. In addition to the new Student Developer Pack partners you may have seen and the fresh Octernship opportunities recently posted, there’s more new features on Global Campus. 

Introducing [Learning Paths](https://education.github.com/) on Global Campus, your first steps to building the skills to make the most out of your time using GitHub. Consider this your guide to Learning Paths and how to begin (or continue) your journey on Global Campus.

Starting today you will see one of two Learning Paths at the top of your Global Campus dashboard when you login:

### **If this your first time using GitHub, you will see Get Started With GitHub Learning Path**

![Get Started with GitHub Learning Path](https://github.com/community/community/assets/107881423/2e3118b7-0f50-4598-bffd-3cc8935b4f3e)

[Get Started with GitHub](https://education.github.com/) includes three Experiences that guide you through setting up your Profile README, exploring your new community by creating a repository watchlist with your favorite topics or projects and finally walking you through the basics of how to use Markdown, the markup language for creating formatted text using a plain-text editor in GitHub.

### **If you’ve used GitHub previously, you will see the GitHub Deep Dive Learning Path**

![GitHub Deep Dive Learning Path](https://github.com/community/community/assets/107881423/4301c85f-f6ee-4a0d-bfc3-995ce23e65bb)

[GitHub Deep Dive](https://education.github.com/) includes two Experiences that build on the foundational knowledge of Get Started with GitHub. In this Learning Path you will be led through a step by step tutorial on the basic GitHub Flow. By the end, you will know how to create a branch, commit a file, and how to open and merge pull requests. Next, you will get a primer on how to use Codespaces, our development environment that's hosted in the cloud.

We hope these new Learning Paths are just the beginning of your journey on GitHub and encourage you to further explore your student benefits as a student on Global Campus.

If you have questions or wish to report any bugs with the [Learning Paths](https://education.github.com/) as you’re completing them, you can create a Discussion post below.

**Comment by orazmyradov** [comment]
Thank you for sharing this information about the new Learning Paths on Global Campus. It's great to see GitHub offering structured guidance for both newcomers and returning users. These Learning Paths will undoubtedly help students make the most of their GitHub experience, whether they're starting fresh or looking to deepen their skills. Happy learning! 🚀📚

**Reply by JRIVERADDIAZ** [reply]
helpful

**Comment by Antismart** [comment]
I just started my learning and I'm stack on Communicate with markdown. I cant edit the index.md file. Could I get help with this please.

**Reply by morganersery1** [reply]
Hello @Antismart. Thanks for the note. You won't be able to edit the file until the PR has been created. If you return to that module and go through steps 1 through 4 again, by the time you get to step 5 (editing the file), it should work.

**Comment by Binary-Ninja-007** [comment]
Hi, everybody.

**Comment by Giangpro89** [comment]
5hankyou

**Comment by Deman-71** [comment]
Я впервые в этой теме АTI хочу разобраться в чем суть

**Comment by 0x94t3z** [comment]
interesting!

**Comment by GabTheGreat25** [comment]
Let's Go Guys! Good Luck Everyone!

**Comment by mevan101** [comment]
I am just a beginner. Hoping these Learning paths are helpful

**Comment by AbdulRehmanGHub** [comment]
Very helpful learning paths for beginners. I also completed all of them. Thank you for sharing.

**Reply by 1154619573** [reply]
hello

**Comment by ALVIN-OBONYO** [comment]
I had to learn these paths the hard way, and I still don't know how to write a proper README file. Thank GitHub for this.

**Comment by 0tieno** [comment]
Very very helpful.

**Comment by 0tieno** [comment]
What is this email error message. "All jobs have failed"

**Comment by Emmarie-Ahtunan** [comment]
Nice refreshers, thanks for the opportunity to win some awesome GitHub Mona swag!

**Comment by Emmarie-Ahtunan** [comment]
Workflows are failing with this error/warning, even when all steps are completed correctly: [Autograding](https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-Emmarie-Ahtunan/actions/runs/6142174281/job/16664000740)
The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/

**Comment by LucioD3v** [comment]
I share the URL of the Learning Path: 

[Intro to GitHub Flow](https://education.github.com/experiences/series_intro_github_flow)

**Reply by LucioD3v** [reply]
I'm already done with the Learning Path!

![image](https://github.com/community/community/assets/6353852/d6235fed-2a37-4645-993f-362fb172c41b)

[https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-LucioMSP](https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-LucioMSP)

**Comment by duaazehra43** [comment]
> ![NUX_Octodex](https://user-images.githubusercontent.com/107881423/262129753-57e515c5-5a64-459c-95eb-24609681e12f.gif) [ ![NUX_Octodex](https://user-images.githubusercontent.com/107881423/262129753-57e515c5-5a64-459c-95eb-24609681e12f.gif) ](https://user-images.githubusercontent.com/107881423/262129753-57e515c5-5a64-459c-95eb-24609681e12f.gif) [ ](https://user-images.githubusercontent.com/107881423/262129753-57e515c5-5a64-459c-95eb-24609681e12f.gif)
> 
> ### All About Learning Paths
> New to GitHub and still navigating your first days here on Global Campus? Or maybe you’re returning to start a new semester this fall. In addition to the new Student Developer Pack partners you may have seen and the fresh Octernship opportunities recently posted, there’s more new features on Global Campus.
> 
> Introducing Learning Paths on Global Campus, your first steps to building the skills to make the most out of your time using GitHub. Consider this your guide to Learning Paths and how to begin (or continue) your journey on Global Campus.
> 
> Starting today you will see one of two Learning Paths at the top of your Global Campus dashboard when you login:
> 
> ### **If this your first time using GitHub, you will see Get Started With GitHub Learning Path**
> ![Screenshot 2023-08-18 at 2 25 36 PM](https://user-images.githubusercontent.com/107881423/262129881-2e3118b7-0f50-4598-bffd-3cc8935b4f3e.png)
> 
> Get Started with GitHub includes three Experiences that guide you through setting up your Profile README, exploring your new community by creating a repository watchlist with your favorite topics or projects and finally walking you through the basics of how to use Markdown, the markup language for creating formatted text using a plain-text editor in GitHub.
> 
> ### **If you’ve used GitHub previously, you will see the GitHub Deep Dive Learning Path**
> ![Screenshot 2023-08-18 at 2 26 04 PM](https://user-images.githubusercontent.com/107881423/262130017-4301c85f-f6ee-4a0d-bfc3-995ce23e65bb.png)
> 
> GitHub Deep Dive includes two Experiences that build on the foundational knowledge of Get Started with GitHub. In this Learning Path you will be led through a step by step tutorial on the basic GitHub Flow. By the end, you will know how to create a branch, commit a file, and how to open and merge pull requests. Next, you will get a primer on how to use Codespaces, our development environment that's hosted in the cloud.
> 
> We hope these new Learning Paths are just the beginning of your journey on GitHub and encourage you to further explore your student benefits as a student on Global Campus.
> 
> If you have questions or wish to report any bugs with the Learning Paths as you’re completing them, you can create a Discussion post below.

Thankyou for sharing

**Comment by LucioD3v** [comment]
I share the URL of the Learning Path:

[Primer: Codespaces](https://education.github.com/experiences/primer_codespaces)

**Comment by vivek9758** [comment]
Completed 🎉

**Comment by TauqeerAhmad5201** [comment]
Amazing line-by-line tutorial it is. Nice opportunity for us to learn and rise up with new things. [🚀](https://emojipedia.org/rocket)

**Comment by fanglong-it** [comment]
Thank you for providing this valuable information about GitHub Deep Dive and the Learning Paths on Global Campus. It's great to know that there are opportunities to further develop our skills and knowledge on GitHub. <3

**Comment by dev348** [comment]
Hi, Am Meenakshisundareswaran R 

thank you team for this

**Comment by annuk123** [comment]
Hello 👋 I'm Annu Kumari 

Thank you! for providing valuable content and information 
About Github deep dive and learning paths on global campus.😇😇

**Reply by dev348** [reply]
Nice to meet you , @annuk123

**Reply by annuk123** [reply]
Me too 😇

**Comment by selcukemiravci** [comment]
**Hi there!**

This was a very cool workflow, I also think a workflow utilizing issues would be super cool as most open source projects use ticketing. It would help new developers get familiar with the process. 

Additionally couple of _feedback_ from my side:

I think for the first learning path where you do a merge and delete the temp branch, this process can be definitely broken down into fewer steps. I think the guide makes it a bit more over complicated however it is still a major requirement for new developers! I really enjoyed how github actions automatically updates the readme every time you complete an assignment.

For the codespaces tutorial, I believe it was great. I didn't use codespaces before so maybe something to think for the team is having a small gif about the main steps on the setup process. This would significantly make it more clear for beginners and more easy to follow through. But definitely an amazing job from the team 👏  💯  

The steps I was referring earlier:

```
Before creating the Codespace, enable GitHub Copilot for your account. If it is not enabled, take a look at [Customize your portfolio site using Copilot](https://github.com/selcukemiravci/codespaces-testt#-customize-your-site-in-3-steps).

Navigate to the main page of the newly created repository.

Under the repository name, use the Code drop-down menu, and in the Codespaces tab, select "Create codespace on main".

Create codespace

Wait as GitHub initializes the Codespace.
```

**Comment by LeonYew-Ley** [comment]
Great tutorial, which is as simple as Google Android Beginner Lab that I've learned before on the last semester.

**Comment by dunalpasindu** [comment]
Thank you!

**Comment by Angell4S** [comment]
Gracias!

**Comment by elviraIv** [comment]
The opportunity to learn and grow is a gift, and I am deeply appreciative of the resources and support that have been provided. Thank you for fostering an environment that encourages continuous learning and intellectual curiosity.

**Comment by eleanarey** [comment]
Is complete and easy to start

**Comment by celesteamador** [comment]
Hi I'm having a problem with my progress in the Exp-Intro-to-GitHub-Flow-Cohort-1
I left the repository because the autograde was not working and I don't know how to rejoin. Whenever I go back to the classroom in the GitHub Education it marks an issue with the assignment. This is the ropository: https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-celesteamador

<img width="731" alt="Screen Shot 2024-01-23 at 15 28 21" src="https://github.com/community/community/assets/110855737/ec75fea6-e351-4bcd-b0ea-d88d951aa608">

**Reply by Liwei254** [reply]
I have the same issue,,,,, did you get help?

**Comment by Dapjeongneo-aysenur** [comment]
Hi, I stack "Click New pull request, for the branches to compare, select base: main and compare: start-markdown." part. 
I cannot find a branch call as start-markdown. 
![image](https://github.com/community/community/assets/94196503/c13bdd99-1022-4eee-ab29-3bb3c4624c32)
What I do wrong:.(

**Comment by ALVIN-OBONYO** [comment]
No I gave up
